src/components/v5/frame/NavigationSidebar/partials/NavigationSidebarMainMenu/types.ts

```diff
@@ -4,11 +4,14 @@ import { NavigationSidebarThirdLevelProps } from '../NavigationSidebarThirdLevel
 
 export interface NavigationSidebarItem extends NavigationSidebarButtonProps {
   key: string;
-  secondLevelMenuProps: Omit<
+  secondLevelMenuProps?: Omit<
     NavigationSidebarSecondLevelProps,
     'isExpanded' | 'onArrowClick'
   >;
   relatedActionsProps?: NavigationSidebarThirdLevelProps;
+  onClick?: () => void;
```

Comment:
onClick and className should already exist in `NavigationSidebarButtonProps` interface, so you don't have to add it here